hedera-node/hedera-schedule-service-impl/src/main/java/com/hedera/node/app/service/schedule/impl/ScheduleStoreUtility.java

```diff
@@ -59,13 +59,6 @@ private static void addToHash(final Hasher hasher, final Key keyToAdd) {
         hasher.putBytes(keyBytes);
     }
 
-    @SuppressWarnings("UnstableApiUsage")
```

Comment:
why do you delete it?

Reply:
It wasn't being used anywhere. Since it uses AccountID, it doesn't really belong in schedule service anyway. My guess is that it was only there due to a copy-pasta mistake.